public/character/animations.json

```diff
@@ -0,0 +1,30 @@
+{
+  "main": {
+    "kibi": {
+      "clips": [
+        "context",
+        "failure",
+        "jump",
+        "success2_l",
```

Comment:
is this notation intentional?

Reply:
not really, it's just the filename the animation had out of the box, it would probably be best to rename it though!